### PR TITLE
Clear quantity positions

### PR DESCRIPTION
**Pull request by hieronymusma**
Reset all quantity positions :)

### CODE REVIEW

**Comment by TomaT3** [comment]
Change httpget to httpput

**Comment by hieronymusma** [comment]
Finished :)

File: SpiritSpenderServer/SpiritSpenderServer/Controllers/ShotGlassPositionsController.cs [hunk 1]
```diff
@@ -33,6 +33,16 @@ public async Task<ActionResult<long>> GetNumberOfPositions()
             return new ObjectResult(numberOfPositions);
         }
 
+        [HttpPut("clear")]
+        public async Task<ActionResult<IEnumerable<ShotGlassPositionSetting>>> GetClearPositions()
```
Comment (TomaT3) [inline SpiritSpenderServer/SpiritSpenderServer/Controllers/ShotGlassPositionsController.cs:37]: Don't you also want to change the method name. No "get" needed

Comment (hieronymusma) [reply SpiritSpenderServer/SpiritSpenderServer/Controllers/ShotGlassPositionsController.cs]: Dont be so picky

**Pull request merged**